### PR TITLE
okapi-testing with UtilityClassTester OKAPI-999

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

File: okapi-testing/src/main/java/org/folio/okapi/testing/UtilityClassTester.java [hunk 1]
```diff
@@ -18,15 +18,15 @@ private UtilityClassTester() {
...
-                       This is save because
-                       - it runs during unit tests only, and
-                       - it invokes the constructor only, no write access, no i/o.
-                       There is no other way to test whether a private constructor
-                       throws an Exception.
-                     */
+      "squid:S1166",  /* suppress "Either log or rethrow this exception" */
+      "squid:S3011",  /* suppress "Changing accessibility is security sensitive" and
+                         suppress "Make sure that this accessibility update is safe here."
+                         This is save because
```
Comment (hjiebsco) [inline okapi-testing/src/main/java/org/folio/okapi/testing/UtilityClassTester.java:24]: typo `save`

File: okapi-testing/src/test/java/org/folio/okapi/testing/UtilityClassTesterTest.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+import org.junit.Test;
+
+public class UtilityClassTesterTest {
+  @Test
+  public void selfTest() {
+    UtilityClassTester.assertUtilityClass(UtilityClassTester.class);
+  }
+
+  public void assertMessage(Class<?> clazz, String message) {
```
Comment (hjiebsco) [inline okapi-testing/src/test/java/org/folio/okapi/testing/UtilityClassTesterTest.java:15]: Need to be `public`?

File: okapi-testing/src/main/java/org/folio/okapi/testing/UtilityClassTester.java (hunk 1 above)
Comment (adamdickmeiss) [reply okapi-testing/src/main/java/org/folio/okapi/testing/UtilityClassTester.java:24]: yep

File: okapi-testing/src/test/java/org/folio/okapi/testing/UtilityClassTesterTest.java (hunk 2 above)
Comment (adamdickmeiss) [reply okapi-testing/src/test/java/org/folio/okapi/testing/UtilityClassTesterTest.java:15]: private is fine.

**Pull request merged**